### ISSUE TITLE
Flow-Builder Action is not documented correctly

### ISSUE THREAD

**Issue by Nico-Schaefer-2111** [shopware/docs#948]
The Guide, how to implement a flow builder Action is not documented correctly. 
Specifically the part on how to extend the administration seems to be deprecated:

[Add Flow Builder action](https://developer.shopware.com/docs/v/6.4/guides/plugins/plugins/framework/flow/add-flow-builder-action#add-custom-action-in-administration)

**Comment by sushmangupta** [comment]
@Nico-Schaefer-2111 , thanks for raising this issue. An [internal ticket](https://shopware.atlassian.net/browse/NEXT-28215) has been created to improvise the related doc.

**Comment by nguyenky** [comment]
Hi @Nico-Schaefer-2111 ,
I re-checked with the `6.5.xx` platform version, and it still works correctly.
It would be nice if you also give us more information about the platform version you are using, so it'll be easy to support your problem.

**Comment by Nico-Schaefer-2111** [comment]
@nguyenky 
Mhh this is quite a time ago. I don't have the specific example anymore since I had to work around the problem I was facing. 

i was using the Version 6.4.19.0 (I linked the documentation page for 6.4.x as well).
If I remember correctly there was a problem when i tried to implement the action configuration: [Add configuration for action](https://developer.shopware.com/docs/v/6.4/guides/plugins/plugins/framework/flow/add-flow-builder-action#step-2-add-configuration-for-action)
